scripts/IC.py

```diff
@@ -92,6 +94,32 @@ def client_install(comm, command, quit_ssh, device, _):
         command(f'interface gpon {client["frame"]} {client["slot"]}')
         command(f'ont delete {client["port"]} {client["onu_id"]}')
         log("Cliente no se agrego apropiadamente en OLT, eliminando...", "warning")
+
+        (client["device"], client["vendor"]) = type_finder(comm, command, client)
```

Comment:
This section is conditioned by "not install", why add the client when the condition is "not install"?

Reply:
In the condition where the client is not installed, the data is also added in the OLT but not in the DB, making the subsequent management of that user by IT impossible since when searching the record it will return that the client does not exist.